fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/bottomsheet/BottomSheet.kt

```diff
@@ -225,6 +229,9 @@ fun BottomSheet(
     showHandle: Boolean = true,
     slideOver: Boolean = true,
     enableSwipeDismiss: Boolean = false,
+    stickyBehavior: Boolean = false,
+    stickyThresholdUpward: Float? = null,
```

Comment:
lets have 56.dp as default value for both

Reply:
Done